### PR TITLE
Improve noncompliant example for python/pytorch-redundant-softmax@v1.0

### PR DESCRIPTION
**Pull request by ankit-amazon**
*Description of changes:*
*Changed non compliant example for `python/pytorch-redundant-softmax@v1.0`*

### CODE REVIEW

**Review by san2488 — COMMENTED** [review]
This is more complex than what I can review confidently. Please have Linghui take a look. I'll approve if she okays this.

**Comment by linghuiluo** [comment]
This is ok
